module/VuFind/src/VuFind/ILS/Driver/Aleph.php

```diff
@@ -1419,51 +1417,28 @@ public function getMyFines($user)
             $barcode = (string)$z30->{'z30-barcode'};
             $checkout = (string)$z31->{'z31-date'};
             $id = $this->barcodeToID($barcode);
-            $mult = ($transactiontype == "Credit") ? 100 : -100;
+            $mult = (string)($item->attributes()->{'type'} ?? '') == 'debit'
```

Comment:
I notice that the old code checked for "Credit" (with a capital C) while the new code checks for "debit" (all lowercase). Does this indicate that there may be inconsistent casing between Aleph installations? If so, would it be worth putting in a strtolower here just to be on the safe side?

Reply:
Before, the text value of tag was checked, which is in Aleph literally a free text, fully configurable on Aleph's site and even translatable. That's the reason why it mostly didn't work. Now I check the xml attribute, which should be always lower cased... But to be absolutly sure, we could lowercased it. I added a followup.

Reply:
Yes, that makes sense. Thank you! Just one last very minor detail -- you're using `->{'type'}` here, but you can simplify this to `->type`. You only need to use the "braces-and-string" syntax when a key contains characters which confuse the PHP parser... so we need it for the properties with dashes in them, but not for a simple type key.

Reply:
Should be fixed now. Thank for all your feedback @demiankatz